.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
aligning to our standard `.eslintrc` would be good. That version contains additional style and security rules.
https://github.com/ccnmtl/ctldjango/blob/master/eslintrc.js

Reply:
@sdreher Got it, I updated the `.eslintrc` to match.

I'll address the eslint errors in the next pr, along with setting up Travis.